aws_asdi_pipelines/pipelines/amazonia_1/config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+  id: "amazonia1"
```

Comment:
@fredliporace Can you change the config `id` to `amazonia_1`.   My image_builder still naively uses the `id` to [infer the location of the code](https://github.com/developmentseed/aws-asdi-pipelines/blob/main/image_builder.py#L22).  Until I update this, the `id` and the pipeline directory name need to match.

Reply:
Ok, my bad. The docs are clear that the id should be the same as the pipeline's parent folder.